content/en/community/hackathons/usoc23/app-porting/index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "Session 06: App Porting"
+linkTitle: "06. App Porting"
+weight: 6
+---
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/intro.md" markdown="true" >}}
+
+## Requirements and Reminders
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Reminders.md" markdown="true" >}}
+
+## Overview
+
+### Support Files
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Overview_Support-Files.md" markdown="true" >}}
+
+### Git Structure
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Overview_Git-Structure.md" markdown="true" >}}
+
+### Example of External Library
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Overview_Example-of-External-Library.md" markdown="true" >}}
+
+### Warm-Up
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Overview_Warm-Up.md" markdown="true" >}}
+
+## Summary
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Summary.md" markdown="true" >}}
+
+## Practical Work
+
+{{< readfile file="/community/hackathons/sessions/contributing-to-unikraft/content/Practical-Work.md" markdown="true" >}}
+
+### Support Files
```

Comment:
This is the extra `Support Files` section I was talking about, there is an almost identical one above it